contributors/devel/release.md

```diff
@@ -300,20 +302,21 @@ of the issue.
   - even less urgent / critical than `priority/important-soon`
   - moved out of milestone more aggressively than `priority/important-soon`
 
-### Issue Kind Label
+### Issue/PR Kind Label
 
 The issue kind is used to help identify the types of changes going
 into the release over time.  This may allow the release team to
 develop a better understanding of what sorts of issues we would
-miss with a faster release cadence.
+miss with a faster release cadence. These labels may also cause escalation to the correct SIG GitHub team.
```

Comment:
nit: wrap 80

Reply:
also, is this true anymore? I don't think it is as the milestone maintainer is dead

Reply:
I just re-edited what was there before, I'm happy to delete it altogether. It was extremely unclear to me what the original sentence meant.